.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Bug report
+about: Create a bug report to help us improve
+title: "[BUG]"
+labels: bug
+assignees: ''
+
+---
+
+<!--
+Oh no, a bug. It happens. Thanks for reporting an issue with NewPipe
+
+Use this template to notify us if you found a bug.
+
+To make it easier for us to help you please enter detailed information below.
+
+Please note, we only support the latest version of NewPipe and
+master branch. Please make sure to upgrade & recreate the issue on the latest
+version prior to opening an issue. The release page (https://github.com/TeamNewPipe/NewPipe/releases/latest) is a good start, make sure its version is the same as in you app (left sidebar, about)
```

Comment:
I think you can be more precise than just left sidebar, about.
Eg: To check you version, swipe from left to right to open the drawer, click on "About", and you will see your current version.